tests/kafkatest/services/verifiable_producer.py

```diff
@@ -79,8 +79,9 @@ def _worker(self, idx, node):
         node.account.ssh("mkdir -p %s" % VerifiableProducer.PERSISTENT_ROOT, allow_fail=False)
 
         # Create and upload log properties
+        producer_prop_file = self.prop_file
```

Comment:
Yes. Maybe a better way is to implement method prop_file() like in ConsoleConsumer, that would enable overriding and also be consistent with ConsoleConsumer?